unity-ggjj/Assets/Scripts/BGScenes/SceneController.cs

```diff
@@ -5,6 +5,9 @@
 
 public class SceneController : MonoBehaviour, ISceneController
 {
+    [Tooltip("Drag a NarrativeScriptPlaylist here")]
```

Comment:
Nit: Mismatching tooltip text and property class.
```suggestion
    [Tooltip("Drag a DialogueController here")]
```